wazuh/wazuh-kibana-app
# Empty file upload

**Description:**

When trying to import an empty file in rules or decoders, there was no information, it let you upload it but it had no effect, I added a warning to let the user know that he is uploading an empty file and not to let him upload it.

**Test:**

1. Navigate to `Management/rules or decoders`
2. Try uploading an empty file
3. See the warning

**Issue:**

- #4059 

**Screenshot:**

![image](https://user-images.githubusercontent.com/63758389/170724585-0ac2c45e-9f16-4f80-9c56-62edc663966f.png)
